docs/CLI.md

```diff
@@ -3,42 +3,42 @@
 ## `predict` command usage:
 
 ```bash
-sahi predict --source image/file/or/folder --model_path path/to/model --config_path path/to/config
+sahi predict --source image/file/or/folder --model_path path/to/model --model_config_path path/to/config
 ```
 
 will perform sliced inference on default parameters and export the prediction visuals to runs/predict/exp folder.
 
 You can specify additional sliced prediction parameters as:
 
 ```bash
-sahi predict --slice_width 256 --slice_height 256 --overlap_height_ratio 0.1 --overlap_width_ratio 0.1 --conf_thresh 0.25 --source image/file/or/folder --model_path path/to/model --config_path path/to/config
+sahi predict --slice_width 256 --slice_height 256 --overlap_height_ratio 0.1 --overlap_width_ratio 0.1 --model_confidence_threshold 0.25 --source image/file/or/folder --model_path path/to/model --model_config_path path/to/config
 ```
 
 - Specify detection framework as `--model_type mmdet` for MMDetection or `--model_type yolov5` for YOLOv5, to match with your model weight
 
 - Specify postprocess type as `--postprocess_type UNIONMERGE` or `--postprocess_type NMS` to be applied over sliced predictions
 
-- Specify postprocess match metric as `--match_metric IOS` for intersection over smaller area or `--match_metric IOU` for intersection over union
+- Specify postprocess match metric as `--postprocess_match_metric IOS` for intersection over smaller area or `--postprocess_match_metric IOU` for intersection over union
 
-- Specify postprocess match threshold as `--match_thresh 0.5`
+- Specify postprocess match threshold as `--postprocess_match_threshold 0.5`
 
-- Add `--class_agnostic` argument to ignore category ids of the predictions during postprocess (merging/nms)
+- Add `--postprocess_class_agnostic` argument to ignore category ids of the predictions during postprocess (merging/nms)
```

Comment:
hmm this name is elegant I liked it :smile:, but it could be more practical like ` --postprocess-ignore-classes` or `--postprocess-ignore-categories`. Just commenting the thoughts, this one can be kept as well.

Reply:
im using `class-agnostic` since its a common term in yolov5 and mmdetection projects, otherwise you are right there are more obvous choices :)